ipa-core/src/cli/crypto/hybrid_encrypt.rs

```diff
@@ -15,11 +20,23 @@ use crate::{
     },
     config::{KeyRegistries, NetworkConfig},
     error::BoxError,
+    hpke::{KeyRegistry, PublicKeyOnly},
     report::hybrid::{HybridReport, DEFAULT_KEY_ID},
     secret_sharing::IntoShares,
     test_fixture::hybrid::TestHybridRecord,
 };
 
+/// Encryptor takes 3 arguments: `report_id`, helper that the shares must be encrypted towards
+/// and the actual share to encrypt.
```

Comment:
nit: maybe use `report` instead of `share` here, since all the types call the thing that's encrypted a `report`.